Add `MCNC7K2022` article (#7193)
ppy/osu-wiki: wiki/Tournaments/MCNC/7K2022/en.md

8. The top 16 seeded players will advance to the Round of 16 (If have enough players there will be Round of 32).

### Stage instructions

1. Following the Qualifier stage, a Double Elimination stage will be played. This means that the winner continues in the winners' bracket, and the losing player gets moved to the losers' bracket.
2. Players in the losers’ bracket must play 2 matches each weekend starting from the Quarterfinals.
3. Players that lose a match in the losers' bracket get eliminated from the tournament.

### Win conditions

- In the Qualifiers, teams need to place in the top 32 seeded teams in order to advance to the Double Elimination stage.
- In the Round of 32, teams need to win 5 maps to win a match (best of 9).
- In the Round of 16 and Quarterfinals, teams need to win 6 maps to win a match (best of 11).
- In the Semifinals, Finals, and Grand Finals, teams need to win 7 maps to win the match (best of 13).

### Match instructions

1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
    - Room settings are osu!mania, Head-to-Head., Win Condition: 'ScoreV2'. Room name must be "MCNC7K2021: (Red) vs (Blue)".

>     - The player mentioned first in the room name must be the red, the player mentioned second in the room name must be the blue.

Reviewer: this doesn't make sense when team mode set to "Head-to-head"

```suggestion
```

> 2. Each player can ban one beatmap to be selected from the pool. These beatmaps can not be picked by any player in the entire match.

Reviewer: ```suggestion
2. Each player can ban one beatmap from the mappool. These beatmaps cannot be picked by any player in the entire match.
```

3. Beatmap selection will alternate between each player selecting a beatmap in the mappool.

> 4. Each player must use !roll once in #multiplayer.

Reviewer: ```suggestion
4. Each player must use `!roll` once in `#multiplayer`.
```

>     - The winner of the !roll starts picking the first beatmap of the match.

Reviewer: ```suggestion
    - The winner of the `!roll` starts picking the first beatmap of the match.
```

>     - The loser of the !roll starts banning one beatmap, followed by the winner of the !roll to ban a beatmap.

Reviewer: ```suggestion
    - The loser of the `!roll` starts banning one beatmap, followed by the winner of the `!roll` to ban a beatmap.
```

> 5. Players will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a player takes more time than alloted, for the second occurrence: a random map will be chosen from the mappool using !roll X, where X is the number of maps that were neither picked nor banned, excluding the Tiebreaker. Repeat offenders may receive further sanctions from the Tournament Management.

Reviewer: ```suggestion
5. Players will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a player takes more time than alloted more than once, a random map will be chosen from the mappool using `!roll` with the number of maps that were neither picked nor banned, excluding the tiebreaker. Repeat offenders may receive further sanctions from the Tournament Management.
```

> 6. Players will be given at most 3-minute pause for solving unexpected situations.

Reviewer: ```suggestion
6. Players will be given at most a 3-minute timeout for solving unexpected situations.
```

> 7. Results of the Qualifiers Stage will be published via a Statistics sheet.

Reviewer: ```suggestion
7. Qualifier results will be published via a statistics sheet.
```

### Mappool instructions

1. There will be one mappool for each stage, except for the Finals and Grand Finals, which will share the same mappool.
2. Each mappool consists of a fixed amount of maps each stage which will all be played under FreeMod conditions. This means that there is a unique FreeMod bracket.
3. The mappool sizes are as follows:
    - Qualifiers: 5 beatmaps
    - Round of 16: 10 beatmaps
    - Quarterfinals: 12 beatmaps

>     - Semifinals : 14 beatmaps

Reviewer: ```suggestion
    - Semifinals: 14 beatmaps
```

    - Finals: 16 beatmaps
4. Each mappool has one tiebreaker, except for the Qualifiers.

> 5. Possible mod choices for all maps are Hidden, FadeIn, Flashlight, and Mirror.

Reviewer: ```suggestion
5. Possible mod choices for all maps are Hidden, Fade In, Flashlight, and Mirror.
```

6. The tiebreaker will be played under FreeMod conditions.

### Scheduling instructions

> 1. Each stage will be held on a single weekend(include Friday). It could be held on weekdays if each players required, but not recommend.

Reviewer: ```suggestion
1. Each stage will be held on a single weekend (including Friday). It can be held on weekdays if both players require it, but this is not recommended.
```

> 2. The schedule is decided by the players discussing with each other. All match times must be communicated to the Tournament Management two days prior to the actual match. If not informed, Tournament Management will decide the game time at his own discretion. The daily schedule will be released the day before.

Reviewer: ```suggestion
2. The schedule is decided by the players discussing with each other. All match times must be communicated to the Tournament Management two days prior to the actual match. If not informed, the Tournament Management will decide the match time at their own discretion. The daily schedule will be released a day beforehand.
```

> 3. If player need to make a temporary change after decide the game time. Rescheduling can only be considered with the consent of both players, and the scheduled time must be at least 1 day after informing the event party.

Reviewer: ```suggestion
3. If a player needs to make a temporary change after deciding the match time, rescheduling can only be considered with the consent of both players, and the scheduled time must be set to at least one day after informing the Tournament Management.
```
